adminSiteClient/Forms.tsx

```diff
@@ -785,6 +819,75 @@ export class BindAutoString<
     }
 }
 
+/** This text field is for cases where you want to have a text field or text area
```

Comment:
I don't think I fully understand how `BindAutoStringExt` is different from `BindAutoString`. Shouldn't something like 

```tsx
<BindAutoString
	field="subtitle"
	store={grapher}
	auto={grapher.currentSubtitle}
/>
```

do exactly what we need (assuming `BindAutoString` supports text areas)?

Reply:
I tried this first but it was annoying to align the uses that were a bit different in different cases. Also, the BindAutoString class is typed with a bunch of `any`'s because the property access can't be typed precisely enough. The approach with getter and setter functions is trivial to type precisely so I prefer that solution. 

Do you think I should rewrite the other uses and remove BindAutoString?

Reply:
Nah, no need to rewrite. I was just wondering if I was missing something essential here :)